JamGame/Assets/Scripts/Employee/EmployeeImpl.cs

```diff
@@ -76,6 +79,45 @@ private struct AppliedBuff
         [SerializeField]
         private Vector2 keepDistanceFromNextInLine = new(1.0f, 1.5f);
 
+        private void Awake()
+        {
+            appliedBuffs.CollectionChanged += (s, e) =>
+            {
```

Comment:
Make method instead of this lambda and name `s` and `e` `sender` and `event` respectively

Reply:
`s` and `e` are widely used abbreviations in the context of c# events when the lambda is subscribed to event